exercises/crypto-square/crypto_square.py

```diff
@@ -1,2 +1,2 @@
-def encode():
+def encode(plaintext):
```

Comment:
It's common practice to use `snake_case` naming convention in Python, so could you please change it accordingly?

You may find this guide useful - https://google.github.io/styleguide/pyguide.html

Reply:
OK, I fixed it.